Lib/test/test_venv.py

```diff
@@ -360,6 +361,26 @@ def test_multiprocessing(self):
             'pool.terminate()'])
         self.assertEqual(out.strip(), "python".encode())
 
+    def test_deactivate_with_strict_bash_opts(self):
+        bash = shutil.which("bash")
+        if bash is None:
+            self.skipTest("bash required for this test")
+        rmtree(self.env_dir)
+        builder = venv.EnvBuilder(clear=True)
+        builder.create(self.env_dir)
+        activate = os.path.join(self.env_dir, self.bindir, 'activate')
+        test_script = os.path.join(self.env_dir, 'test_strict.sh')
+        with open(test_script, 'w') as f:
+            f.write("""
```

Comment:
Using a multiline string like this is a little jarring (because it breaks indentation). Either move the test script to a global variable, or build the string out of single-line, indented strings (like a few of the other tests do).

Reply:
Thanks. Fixed.